dashboard/db/migrate/20160509143719_migrate_deleted_users_net_promoter_score2015_data.rb

```diff
@@ -0,0 +1,51 @@
+# A migration that transfers the net promoter score survey results in the users
+# table from soft-deleted rows to the survey_results table. Does NOT remove the
+# data from the users table.
+
+class MigrateDeletedUsersNetPromoterScore2015Data < ActiveRecord::Migration
+  def change
+    reversible do |direction|
+      # In the up direction, we create SurveyResult records for soft-deleted
+      # users (non-deleted users were handled in a previous migration) having a
+      # survey2015_value or survey2015_comment. For safety, we do not remove the
+      # information from the users row in this migration.
+      direction.up do
+        ActiveRecord::Base.record_timestamps = false
+        begin
+          User.only_deleted.where("properties LIKE '%survey2015%'").
+              find_each do |user|
```

Comment:
Yep, we follow Option B here (https://github.com/bbatsov/ruby-style-guide) [see "Adopt a consistent multi-line method chaining style"], per our rubocop (https://github.com/code-dot-org/code-dot-org/blob/staging/.rubocop.yml#L22).